awesome/src/lua/mod.rs

```diff
@@ -52,12 +37,26 @@ thread_local! {
 ///
 /// Best way to help out: comment out one of these lines, fix what breaks.
 fn load_shims(lua: &Lua) {
+    let shims_path: PathBuf = std::env::current_exe().unwrap().parent()
+        .ok_or(()).unwrap().join("../../tests/awesome/tests/examples/shims/");
+    /// Shims to load
+    const SHIMS: [&str; 11] = ["awesome",
```

Comment:
Please keep this at the top of the file, where it was before.

Reply:
But it's confusing. When it's in the top, it looks like something important, so you need to find all usages of it if you debug something relevant to it.

Whereas when the scope is limited to where it's used, it's obvious that it's a one-time constant.

Reply:
It is important, its all the shims that we still need for Way Cooler to "work". 

Any editor worth its salt will have a "jump to definition" that is language agnostic (or you can try to get the Rust Language Server working, good luck) so it shouldn't be hard to look at the definition if you need it.

Finally, I didn't detail it in the style guide but constants should go at the top. That' standard in most language default "idiomatic" style which is why I forgot it.

Reply:
Ok, I'll change it. That said, you misunderstood me, I didn't mean that it's hard to find the definition. I meant that it's easier to understand the codebase when stuff has as limited visibility scope as possible, because you don't need to check every place where something is used.

But okay, I'll count it as a specific of that particular project, and will stick to that style.